controllers/configmap_controller.go

```diff
@@ -67,9 +67,16 @@ const (
 	ConfigMapController = "configmap"
 )
 
+// newServiceConfig returns a 'Data' object which contains the expected Windows service configuration based on
+// the state of the cluster
+func newServiceConfig() (*servicescm.Data, error) {
+	return servicescm.NewData(&[]servicescm.Service{}, &[]servicescm.FileInfo{})
+}
+
 // ConfigMapReconciler reconciles a ConfigMap object
 type ConfigMapReconciler struct {
 	instanceReconciler
+	serviceConfig *servicescm.Data
```

Comment:
Why is this called `serviceConfig` instead of `servicesData` or even better something like `servicesManifest`?